src/Microsoft.Data/DataFrame.BinaryOperations.cs

```diff
@@ -26,12 +26,15 @@ public DataFrame Add<T>(IReadOnlyList<T> values, bool inPlace = false)
             for (int i = 0; i < Columns.Count; i++)
             {
                 DataFrameColumn baseColumn = _columnCollection[i];
+                baseColumn._reverseOrderOfBinaryOperations = _reverseOrderOfBinaryOperations;
```

Comment:
Adding internal state to `DataFrame` and `PrimitiveDataFrameColumn` seemed to be a cleaner than adding parameters to each binary op.

Reply:
I think I disagree. If an exception is thrown between here and line 35, this state is going to be left to this old value.

---
In reply to: [340205425](https://github.com/dotnet/corefxlab/pull/2769#discussion_r340205425) [](ancestors = 340205425)

Reply:
Also, how could someone write their own derived column class? They won't be able to get access to this internal state.

Can we find some other way to enable this?

---
In reply to: [340322971](https://github.com/dotnet/corefxlab/pull/2769#discussion_r340322971) [](ancestors = 340322971,340205425)

Reply:
For the 2nd question, I thought of that and put the flag on the base class :)

I think I'm going to make new `Radd`, `Rdivide` etc virtual methods. That should fix all our concerns

Reply:
The method names should follow the .NET guidelines though - 

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines